README.md

```diff
@@ -22,19 +22,208 @@ Our main objective is to provide the RSK ecosystem with the means to enable bloc
+  "gasPriceFactor": 1,
+  "rskNodeUrl": "http://rsk-node:4444",
+  "devMode": true,
+  "customReplenish": false,
+  "logLevel": 1,
+  "workdir": "/home/jonathan/workspace/enveloping/environment"
+}
+```
+
+Where:
+
+* **url**: is the url where the relay server will be deployed, it could be localhost or the ip of the host machine
+* **port**: as url is the port where the relay server will be hosted.
+* **relayHubAddress**: is the relay hub contract address, you can retrieve this from the contract summary.
+* **relayVerifierAddress**: is the relay verifier contract address, you can retrieve this from the contract summary.
+* **deployVerifierAddress**: is the deploy verifier contract address, you can retrieve this from the contract summary.
+* **gasPriceFactor**: is the gas price factor used to calculate the gas on the server, you can leave it as 1.
+* **rskNodeUrl**: is the RSK node endpoint url, where the RSK node is located.
+* **devMode**: it indicates to the server if we are in development mode or not.
+* **customReplenish**: (Pending documentation)
+* **logLevel**: is the log level for the relay server.
+* **workdir**: is the absolute path to the folder where the server will store the database and all it's data.
+
+3. Now we can use the command `yarn relay` (on the root of the enveloping project) to start the relay server.
+After running that command you will see a log saying that the relay server is not ready and that some values are wrong, that's ok you just need to register this relay server into the relay hub in order to be usable by the clients.
+
+## Register the Relay Server
+
+### On Regtest
+
+The relay server is running, now you need to register this server in order to be usable, to do so you
+first need to configure the script located on `<PROJECT_ROOT>/scripts/registerRelayServer` and replace the 
+   values as you consider, the script contains something like this:
+
+```
+node dist/src/cli/commands/enveloping.js relayer-register --funds 100 --stake 200 --network http://rsk-node:4444/ --hub "0x3bA95e1cccd397b5124BcdCC5bf0952114E6A701"
+```
+
+Where:
+
+* **--fund**: indicates the amount of rbtc that you will transfer from accounts[0] to the worker manager account.
+* **--stake**: (Pending documentation) this should be twice the value of funds.
+* **--hub**: is the relay hub contract address, you can retrieve this from the contract summary.
+
+After doing that you need to open another terminal and run this command `yarn registerRelay` \
+(on the root of the enveloping project) in order to register the relay. After running that command you will
+be seeing some logs saying that everything executes correctly. Then if you go to the relay server
+logs you will be seing a lot of interaction but you need to see this log to be sure the server is ready:
+
+```
+Relayer state: READY
+```
+
+### On Testnet
+
+1. Send to relayManagerAddress at least 0.001 tRBTC to set it up
+2. Send to relayWorkerAddress at least 0.001 tRBTC to set it up
```

Comment:
we should indicate how to obtain these addresses, (i.e. go to `http://localhost:8080/`or whatever the URL is)

Reply:
agree